registration/registration-test/README.md

```
@@ -44,6 +44,9 @@ Update the following:
 1. Verify the failure in the logs file `\logs\AutomationLogs.log`
 1. Execution results present under report folder file `extentReport-datetimestamp.html`
 
+## Automated Test Cases
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)